main.go

```diff
@@ -386,7 +386,8 @@ All long form (--) flags can be toggled with the dig-standard +[no]flag notation
 			!strings.Contains(arg, "@") &&
 			!strings.Contains(arg, "/") &&
 			!strings.HasPrefix(arg, "-") &&
-			!strings.HasPrefix(arg, "+") {
+			!strings.HasPrefix(arg, "+") &&
+			!strings.Contains(arg, ".exe") {
```

Comment:
Can we change this `.exe` to a `\` backslash character? I think it should accomplish the same result and won't depend on the file having an exe extension.

Reply:
I did some testing and that check would actually parse the full path (including backslashes, which later get filtered out) so this would've worked, however when testing in cmd it sent `q.exe` as the queryname instead of the domain, so we're back to the other solution I think. It is now cmd messing up instead of powershell lol.

Seeing as this is a Windows only issue (as far as we know and have been able to test at least) I think that checking for `.exe` might be the best solution.

What do you think?

Reply:
Got it, that makes sense. LGTM!